Makefile

```diff
@@ -32,9 +39,15 @@ ta:
 			  $@
 
 .PHONY: clean
+ifneq ($(wildcard $(TA_DEV_KIT_DIR)/host_include/conf.mk),)
 clean:
 	$(q)$(MAKE) -C host/xtest O=$(out-dir)/xtest q=$(q) $@
 	$(q)$(MAKE) -C ta O=$(out-dir)/ta q=$(q) $@
+else
+clean:
+	$(q)echo "TA_DEV_KIT_DIR is not correctly defined - Remove $(out-dir)"
+	$(q)rm -rf $(out-dir)
```

Comment:
Right, it's too dangerous.
I'll check if the clean can work when the dev_kit is not there (it may be possible). Or if not possible, we'll just have an echo || false, as in the `all:` target